plugin.json

```diff
@@ -0,0 +1,36 @@
+{
+  "id": "marketplace-addon",
```

Comment:
Let's call this: `com.mattermost.marketplace-addon' for now.

Reply:
done by cea605cb5656384b9336f2d6a311ec775249d59a